src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -1440,6 +1441,40 @@ def perform_destroy(self, instance):
         },
     )
 )
+@extend_schema_view(
+    create=extend_schema(
+        description=(
+            "Reserveer een of meerdere zaaknummers.\n\n"
+            "Als `aantal` niet wordt opgegeven of gelijk is aan 1, wordt een enkel object teruggegeven.\n"
+            "Als `aantal > 1`, wordt een lijst van objecten teruggegeven."
+        ),
+        responses={
+            201: {
+                "oneOf": [
+                    ReserveZaakIdentificatieSerializer,
+                    ReserveZaakIdentificatieSerializer(many=True),
+                ]
+            }
```

Comment:
This isn't showing up in the schema currently because of the duplicate `extend_schema_view` decorators, if you join them into one and do this I think it will show properly (using `drf_spectacular.utils.PolymorphicProxySerializer`)

```suggestion
            status.HTTP_201_CREATED: OpenApiResponse(
                response=PolymorphicProxySerializer(
                    component_name="OneOrMultipleZaakIdentificaties",
                    serializers=[
                        ReserveZaakIdentificatieSerializer,
                        ReserveZaakIdentificatieSerializer(many=True),
                    ],
                    resource_type_field_name=None,
                    many=False
                )
            ),
```

Will show up like:

![afbeelding](https://github.com/user-attachments/assets/18017ea1-ddeb-4301-99ac-e04e3c7b1859)